tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -130,6 +130,8 @@ type Stresser interface {
 	Cancel()
 	// Report reports the success and failure of the stress test
 	Report() (success int, failure int)
+	// Setup initializes fields of underlying Stresser struct
+	Setup() error
```

Comment:
Please don't add this to the interface. The stresser can know when Stress() is first called and do any initialization at that point, there's no need to have a separate Setup function at the interface level yet.